src/physics/autompo.jl

```diff
@@ -146,9 +152,13 @@ function Base.show(io::IO, op::MPOTerm)
   end
   for o in ops(op)
     print(io, "\"$(name(o))\"($(string_site_or_sites(o))) ")
+    !isempty(params(o)) && print(io, params(o))
   end
 end
 
+*(α::Number, op::MPOTerm) = MPOTerm(α * coef(op), ops(op))
+*(op::MPOTerm, α::Number) = α * op
```

Comment:
Could you add:
```julia
/(op::MPOTerm, α::Number) = MPOTerm(coef(op) / α, ops(op))
```
as well? I came across that for Trotterization.

Also a neat thing is that you can use the notation:
```julia
(α::Number * op::MPOTerm) = MPOTerm(α * coef(op), ops(op))
```